README.md

```diff
@@ -2,37 +2,39 @@
 
 ## About this task
 
-Think of this as an open source project. How would this have to look, in order for you to be impressed with it if you were to find it on Github? Now go do that.
+Think of this as an open source project. How would this have to look in order for you to be impressed with it, if you were to find it on GitHub? Now go and do that.
 
-Try to limit the amount of time you spend on this to 90-120 minutes. However, feel free to spend more - just make sure you're happy with your submission!
+Try to limit the amount of time you spend on this to 90-120 minutes. Feel free to spend more time - make sure you are happy with your submission!
 
-_Hint_: we're looking for a high-quality submission with great application architecture, not a "just get it done"-like approach. Stay away from frameworks/boilerplates that handle everything for you - or use them only as a thin layer - so we can see how you structure applications yourself.  Remember that this test is your opportuniity to show us how you think, so be clear about how you're thinking in your code - whether that's with comments, tests, how you name things etc.
+_Hint_: we are looking for a high-quality submission with great application architecture. Not a "get it done" approach. Stay away from frameworks and boilerplates that handle everything for you. If you do use a framework, only use it as a thin layer so we can see how you structure applications yourself.  
+
+Remember that this test is your opportunity to show us how you think. Be clear about how you make decisions in your code, wether that is with comments, tests or how you name things.
```

Comment:
A `wether` is a castrated ram. Unless we've changed direction massively, we probably still want `whether`.

I'm also not the biggest fan of the Oxford comma, but I think in this case the final sentence reads a little better if you add once before the final `or`

Reply:
Today I learned! Updated and added an extra comma.